src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.DgmlHelper/xlf/Resources.cs.xlf

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../Resources.resx">
+    <body>
+      <trans-unit id="SyntaxNodeLabel">
+        <source>'{0}' Node</source>
+        <target state="new">'{0}' Node</target>
```

Comment:
I was wondering if just the Token one should be quoted (and not the Node), because the text there is the actual text found in the code. Up to you though.

Reply:
Yes, that would be better. I fixed it in the newest version. Thanks!